drafts/use-cases/1.1.security-considerations.md

```diff
@@ -0,0 +1,40 @@
+### Security considerations
+
+#### JSON array root security issues
+Due to security issues with JSON arrays which can be exploited in some circumstances, 
+there may be a need to avoid pure arrays in favour of a JSON object.
+In the scenario above, server returned a single object, which won't cause issues.
+But it is possible to return multiple resources, resulting in an JSON array returned 
+(server may not use any JSON-LD context or it may be completely different from the one used by the client).
+There are several possibilities here. One of them would be to use a *@graph* notation, i.e.:
+
+```javascript
+{
+    "@graph": [
+        {
+            "@id": "/api",
+            "operations": [
+                {
+                    "@id": "/api/events",
```

Comment:
Given `/api/events` denotes the operation `"@type": "hydra:Operation"`, would some different URI would denote the resource `"@type": "hydra:Collection` on which client can perform this operation? e.g `/data/events`